list/List.js

```diff
@@ -182,6 +182,8 @@ define([
 
 		template: template,
 
+		tabindex: 0,
```

Comment:
Potentially I may rename the `tabIndex` property into something else, like `_tabIndexAttr` and bind it into the template. Makes sense to you?